js/apps/events-page/controllers/eventsPageCtrl.js

```diff
@@ -6,16 +6,16 @@
 		$timeout,
 		$animate,
 		$location,
+		$window,
 		CONSTANTS,
 		eventsService,
-		filterHelperService, 
+		filterHelperService,
 		metaService
 	) {
-
 		const vm = this;
 		const firstPage = 1;
-		const startDateLocaleString = moment().format();
-		const endDate = moment().add(30, 'd');
+		const startDateLocaleString = $window.moment().format();
```

Comment:
This isn't a blocking comment, i think there is a pattern to pass in non angular libraries into an angular app. Don't have time to find, but I think the general idea is to create a factory to determine if the library exists on the window, and then you can use angulars normal dependency inject. Probably not worth it once we switch to react 👐